tests/site/long_page.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Long Page</title>
+    <script src="//code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script>
+        $(document).ready(function() {
+            $("#fixture input").click(function(theEvent) {
+                $("#output").html("button was clicked");
+            });
+        });
+    </script>
+
+</head>
+<body>
+    <p id="long_part"></p>
+    <p id="element_after_long_part">scroll here</p>
+    <p id="next_long_part"></p>
+    <script>
```

Comment:
Is there a reason these scripts are here instead of in the head element?
It might be a more straightforward to set the heights of long parts directly.